spotbugs/src/main/java/edu/umd/cs/findbugs/PriorityAdjuster.java

```diff
@@ -79,11 +86,42 @@ BugInstance adjustPriority(BugInstance bugInstance) {
         if (patternAdjustment != null) {
             priority = patternAdjustment.adjust(priority);
         }
+        priority = BugInstance.boundedPriority(priority);
         if (priority == bugInstance.getPriority()) {
             return bugInstance;
         }
         BugInstance clone = ((BugInstance) bugInstance.clone());
         clone.setPriority(priority);
         return clone;
     }
+
+    /**
+     * Adjust priority if the factory was forcibly enabled by {@link ExecutionPlan} although the user did not select it.
+     *
+     * @param bugInstance
+     * @return adjusted bug priority
+     */
+    public int adjustForDetector(BugInstance bugInstance) {
+        DetectorFactory factory = bugInstance.getDetectorFactory();
+        if (factory != null) {
+            if (factoryChooser != null && !factoryChooser.wasFactoryEnabled(factory)) {
```

Comment:
Could be combined with the previous condition.

Reply:
Done